src/thumbnailFactory.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var ImageResource = require('./ImageResource');
+
+var _getThumbService = function(image, width) {
+  if(image.resource.service) {
+    return image.resource.service['@id'] + '/full/' + Math.ceil(width / 4) + ',/0/default.jpg';
+  } else {
+    return image.resource['@id'];
```

Comment:
Maybe what I could do here, to address one of your comments in #102 is if there's only one image in the canvas and it's static, I could build a config of imageType 'main' and check for that in the CanvasObject when it gets the thumbnail back. Then the CanvasObject could just not bother opening the real tilesource.